fuzz/fuzzJsonWeb.go

```diff
@@ -0,0 +1,24 @@
+// +build gofuzz
```

Comment:
Is it possible to move this file into `internal/fuzz/jsonweb/jsonweb.go` and change the fuzzing function name to the standard `Fuzz`? I think it will make it easier to use.

Reply:
That should work; I'll test it out. Do you think in that case it might be better to promote the targets to live in package directories themselves (as opposed to shadowing them in `internal`)? 🤔